floris/parallel_floris_model_2.py

```diff
@@ -0,0 +1,309 @@
+
+            for fm in self._fmodels_split[1:]:
+                self.core.flow_field.u = np.append(
+                    self.core.flow_field.u,
+                    fm.core.flow_field.u,
+                    axis=0
+                )
+                self.core.flow_field.v = np.append(
+                    self.core.flow_field.v,
+                    fm.core.flow_field.v,
+                    axis=0
+                )
+                self.core.flow_field.w = np.append(
+                    self.core.flow_field.w,
+                    fm.core.flow_field.w,
+                    axis=0
+                )
+                self.core.flow_field.turbulence_intensity_field = np.append(
+                    self.core.flow_field.turbulence_intensity_field,
+                    fm.core.flow_field.turbulence_intensity_field,
+                    axis=0
+                )
+
+    def _get_turbine_powers(self):
+        """
+        Calculates the power at each turbine in the wind farm.
+        This override will only be necessary if we want to be able to
+        use the return_turbine_powers_only option. Need to check if that
+        makes a significant speed difference.
+
+        Returns:
+            NDArrayFloat: Powers at each turbine.
+        """
+        if self.core.state is not State.USED:
+            self.logger.warning(
+                f"Please call `{self.__class__.__name__}.run` before computing"
+                " turbine powers. In future versions, an explicit run() call will"
+                "be required."
+            )
+            self.run()
+        if self.return_turbine_powers_only:
+            return self._stored_turbine_powers
+        else:
+            return super()._get_turbine_powers()
+
+    @property
+    def fmodel(self):
+        """
+        Raise deprecation warning.
+        """
+        self.logger.warning(
+            "ParallelFlorisModel no longer contains `fmodel` as an attribute "
+            "and now directly inherits from FlorisModel. Please use the "
+            "attributes and methods of FlorisModel directly."
+        )
+
+
+def _parallel_run(fmodel_dict, set_kwargs) -> FlorisModel:
+    """
+    Run the FLORIS model in parallel.
+
+    Args:
+        fmodel: The FLORIS model to run.
+        set_kwargs: Additional keyword arguments to pass to fmodel.set().
+    """
+    fmodel = FlorisModel(fmodel_dict)
+    fmodel.set(**set_kwargs)
+    fmodel.run()
+    return fmodel
+
+def _parallel_run_powers_only(fmodel_dict, set_kwargs) -> np.ndarray:
+    """
+    Run the FLORIS model in parallel, returning only the turbine powers.
+
+    Args:
+        fmodel: The FLORIS model to run.
+        set_kwargs: Additional keyword arguments to pass to fmodel.set().
+    """
+    fmodel = FlorisModel(fmodel_dict)
+    fmodel.set(**set_kwargs)
+    fmodel.run()
+    return fmodel.get_turbine_powers()
+
+def _optimize_yaw_angles_serial(
```

Comment:
Leaning toward not including this function

Reply:
Ah, sorry, my mistake---I didn't mean to commit that. Will remove.